demo/site/src/components/common/CookiePlaceholders.tsx

```diff
@@ -0,0 +1,32 @@
+import { useCookieApi } from "@comet/cms-site";
+import styled from "styled-components";
+
+type Props = {
+    variant: "loading" | "fallback";
+};
+
+export const CookiePlaceholder = ({ variant }: Props) => {
+    const { openCookieSettings } = useCookieApi();
+    return (
+        <Root>
+            {variant === "loading" && <h4>Loading cookie provider...</h4>}
```

Comment:
In a real application, this would probably render `null`, right?

Reply:
Optimally this would be a skeleton-type of element. 
Something that has the height of the element that should be loaded to prevent layout-shift when the provider is initialized.